runelite-client/src/main/java/net/runelite/client/plugins/itemstats/ItemStatPlugin.java

```diff
@@ -67,5 +112,323 @@ protected void startUp() throws Exception
+		);
+
+		int index = 0;
+
+		for (final String stat : stats)
+		{
+			yPos += TEXT_HEIGHT + 2;
+
+			// Style label
+			final Widget styleText = createText(invContainer, stat, FontID.PLAIN_11, ORANGE_TEXT,
+				0, yPos, invContainer.getWidth(), -1);
+			styleText.setXTextAlignment(WidgetTextAlignment.CENTER);
+
+			// Attack bonus
+			createText(invContainer, attackStats.get(index).toString(), FontID.PLAIN_11, YELLOW_TEXT,
+				5, yPos, 50, -1);
+
+			// Defence bonus
+			final int defenceX = invContainer.getWidth() - (smallFM.stringWidth(defenceStats.get(index).toString()) + 5);
+			createText(invContainer, defenceStats.get(index).toString(), FontID.PLAIN_11, YELLOW_TEXT,
+				defenceX, yPos, 50, -1);
+
+			index++;
+		}
+
+		// MISC BONUSES
+
+		yPos += TEXT_HEIGHT + 8;
+
+		final Map<String, Integer> miscStats = ImmutableMap.of(
+			"Strength", equipmentStats.getStr(),
+			"Ranged Strength", equipmentStats.getRstr(),
+			"Magic Damage", equipmentStats.getMdmg(),
+			"Prayer Bonus", equipmentStats.getPrayer()
+		);
+
+		for (final Map.Entry<String, Integer> miscStat : miscStats.entrySet())
+		{
+			final String name = miscStat.getKey();
+			final String value = miscStat.getValue().toString();
+
+			// Stat label
+			createText(invContainer, name, FontID.PLAIN_11, ORANGE_TEXT, 5, yPos, 50, -1);
+
+			// Stat bonus
+			int valueXPos = invContainer.getWidth() - (smallFM.stringWidth(value) + 5);
+			createText(invContainer, value, FontID.PLAIN_11, YELLOW_TEXT, valueXPos, yPos, 50, -1);
+
+			yPos += TEXT_HEIGHT + 2;
+		}
+
+		// COINS
+
+		createSeparator(invContainer, invContainer.getHeight() - 40);
+
+		final String coinText = "You have " + StackFormatter.quantityToRSStackSize(getCurrentGP())
+			+ (getCurrentGP() == 1 ? " coin." : " coins.");
+
+		final Widget coinWidget = createText(invContainer, coinText, FontID.PLAIN_12, ORANGE_TEXT,
+			0, invContainer.getHeight() - 18, invContainer.getWidth(), -1);
+
+		coinWidget.setXTextAlignment(WidgetTextAlignment.CENTER);
+	}
+
+	private static Widget createText(Widget parent, String text, int fontId, int textColor,
+								int x, int y, int width, int height)
+	{
+		final Widget widget = parent.createChild(-1, WidgetType.TEXT);
+		widget.setText(text);
+		widget.setFontId(fontId);
+		widget.setTextColor(textColor);
+		widget.setTextShadowed(true);
+		widget.setOriginalHeight(height == -1 ? TEXT_HEIGHT : height);
+		widget.setOriginalWidth(width);
+		widget.setOriginalY(y);
+		widget.setOriginalX(x);
+		widget.revalidate();
+		return widget;
+	}
+
+	private static void createSeparator(Widget parent, int y)
+	{
+		Widget separator = parent.createChild(-1, WidgetType.GRAPHIC);
+		separator.setOriginalWidth(parent.getWidth());
+		separator.setOriginalY(y);
+		separator.setOriginalHeight(32);
+		separator.setSpriteId(SpriteID.UNKNOWN_BORDER_EDGE_HORIZONTAL_995);
+		separator.revalidate();
+	}
+
+	private void showGEInventory()
+	{
+		final Widget invContainer = client.isResized() ?
```

Comment:
Is this intentionally disregarding `RESIZABLE_VIEWPORT_BOTTOM_LINE_INVENTORY_CONTAINER`?